File: backend/job-application-service/src/main/java/com/jobportal/jobapp/entities/JobStatus.java
package com.jobportal.jobapp.entities;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "job_status")
public class JobStatus {

    @Id
    private Integer jsid;

    @Column(length = 50)
    private String status;

    public JobStatus() {
    }

    public JobStatus(Integer jsid, String status) {
        this.jsid = jsid;
        this.status = status;
    }

    public Integer getJsid() {
        return jsid;
    }

    public void setJsid(Integer jsid) {
        this.jsid = jsid;
    }

    public String getStatus() {
        return status;
    }

    public void setStatus(String status) {
        this.status = status;
    }
}

File: backend-dotnet/Services/JobPortal.JobApplicationService/Entities/JobStatus.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace JobPortal.JobApplications.Entities;

[Table("job_status")]
public class JobStatus
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.None)]
    [Column("jsid")]
    public int Jsid { get; set; }

    [MaxLength(50)]
    [Column("status")]
    public string? Status { get; set; }

    public JobStatus() { }

    public JobStatus(int jsid, string? status)
    {
        Jsid = jsid;
        Status = status;
    }
}
